src/components/MapSelectors/GlobalSelector.js

```diff
@@ -158,8 +158,15 @@ const GlobalSelector = ({ handleChange, extendRight }) => {
     }
 
     if (location && !inputValue) {
+      // the URL location param used to be the stateCode but now is stateName-stateCode
+      // so we now need to grab the stateCode from the end of the param to populate the search dropdown
+      // fixes bug: https://trello.com/c/TfH1UiqD/572-search-dropdown-doesnt-populate-with-state-counties-using-new-url-format
+      const stateCodeFromParam = location.slice(-2);
```

Comment:
you can use `getStateByUrlName` and `getCountyByUrlName` instead, see https://github.com/covid-projections/covid-projections/blob/3fe0bec60ebdd64e92988909bf9822a8baf9153e/src/screens/LocationPage/LocationPage.js#L21-L24